src/App.jsx

```diff
@@ -1,23 +1,41 @@
+import { useEffect, useState } from "react";
 import "./App.css";
 import ProductCard from "./components/ProductCard/ProductCard";
 import CalendarToggle from "./components/calendarToggle/CalendarToggle";
 import PersonProfile from "./components/personProfile/PersonProfile";
+import axios from "axios";
 
 function App() {
+  const [bakendMessage, setBakendMessage] = useState("");
```

Comment:
just a small typo bakend.. am sure you meant backend:)

otherwise good stuff!we already have a backend!!